crates/gitql-ast/src/function.rs

```diff
@@ -223,6 +224,13 @@ lazy_static! {
                 result: DataType::DateTime,
             },
         );
+        map.insert(
+            "now",
+            Prototype {
+                parameters: vec![],
+                result: DataType::Text,
```

Comment:
Now Function expect to return `DateTime` type not `Text` so we can later perform operations for example plus